RDF_Engine/Query/IsDataType.cs

```diff
@@ -36,9 +36,9 @@ namespace BH.Engine.Adapters.RDF
 {
     public static partial class Query
     {
-        public static bool IsDataType(this Type type, TBoxSettings tBoxSettings)
+        public static bool IsDataType(this Type type, ABoxSettings aBoxSettings)
         {
-            return !type.IsOntologyClass(tBoxSettings);
+            return !type.IsOntologyClass(aBoxSettings);
```

Comment:
As mentioned in https://github.com/BHoM/RDF_Prototypes/issues/70#issuecomment-1460310928:
these methods that define the T-Box (IsDataType, IsDataProperty, etc.) should depend on the TBoxSettings, not the ABoxSettings.

Therefore, we should move the GeometryAsOntologyClass into the TBoxSettings.